packages/google_maps_flutter/google_maps_flutter/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
-## NEXT
+## 2.1.6
 
+* Switches to new platform interface versions of `buildView` and
+  `updateOptions`, making it compatible with `google_maps_flutter_web` 0.4.0.
```

Comment:
Capturing from offline discussion: everything works out because the base class forwards the new methods to the old implementations. I've added more discussion to https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changing-platform-interface-method-parameters since it's unintuitive.